web/css/source/variables/overrides/_typography.less

```diff
@@ -2,3 +2,6 @@
 @text__color: @color-text-black;
 @heading__color__base: #000;
 @h1__margin-bottom: @indent__l;
+@message-icon__top: 32px;
```

Comment:
These variables also update the original system page messages, not only your custom ones. is it acceptable?